vms/rpcchainvm/vm_server.go

```diff
@@ -87,14 +86,19 @@ type VMServer struct {
 
 // NewServer returns a vm instance connected to a remote vm instance
 func NewServer(vm block.ChainVM, allowShutdown *utils.Atomic[bool]) *VMServer {
+	pluginMetrics := metrics.NewPrefixGatherer()
+
 	bVM, _ := vm.(block.BuildBlockWithContextChainVM)
 	ssVM, _ := vm.(block.StateSyncableVM)
-	return &VMServer{
+	vmSrv := &VMServer{
 		vm:            vm,
 		bVM:           bVM,
 		ssVM:          ssVM,
 		allowShutdown: allowShutdown,
 	}
+
+	vmSrv.metrics = pluginMetrics
```

Comment:
Can we switch to just inlining this for the smallest diff? ie. add `metrics: metrics.NewPrefixGatherer()` as the next line in the struct definition?